src/ray/rpc/worker/worker_server.h

```diff
@@ -0,0 +1,68 @@
+#ifndef RAY_RPC_WORKER_SERVER_H
+#define RAY_RPC_WORKER_SERVER_H
+
+#include "ray/rpc/grpc_server.h"
+#include "ray/rpc/server_call.h"
+
+#include "src/ray/protobuf/worker.grpc.pb.h"
+#include "src/ray/protobuf/worker.pb.h"
+
+namespace ray {
+namespace rpc {
+
+/// Interface of the `WorkerService`, see `src/ray/protobuf/worker.proto`.
+class WorkerTaskHandler {
+ public:
+  /// Handle a `AssignTask` request.
+  /// The implementation can handle this request asynchronously. When hanling is done, the
+  /// `done_callback` should be called.
+  ///
+  /// \param[in] request The request message.
+  /// \param[out] reply The reply message.
+  /// \param[in] done_callback The callback to be called when the request is done.
+  virtual void HandleAssignTask(const AssignTaskRequest &request, AssignTaskReply *reply,
+                                RequestDoneCallback done_callback) = 0;
+};
+
+/// The `GrpcServer` for `WorkerService`.
+class WorkerTaskGrpcService : public GrpcService {
```

Comment:
I'm confused, where is this used?

Reply:
sorry for the confusion. It's in the next PR which actually changes from worker `GetTask` to raylet `AssignTask`, I will add the missing code to this PR for completeness.